### ISSUE TITLE
[Accordion] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1268]
### Feedback

❌ `aria-label` seems unnecessary on the button as it already has child content of the same text
❌ if panels really can’t be collapsed they should have `aria-disabled="true"` on the button

### Path forward

1. remove `aria-label` usage
2. panels can now close, so we're done on this one.

cc @jnurthen
